src/odemis/acq/feature.py

```diff
@@ -179,6 +180,11 @@ def __init__(self, name: str,
         self.path: str = None  # TODO:support path creation here, rather than on milling data save
         self.reference_image: model.DataArray = None
 
+        # Save the stream used for SuperZ workflow
+        self.superz_stream_name: str = None
+
+        self.superz_focus = None  # True if super z focus has high accuracy, False otherwise
```

Comment:
```suggestion
        self.superz_focus: Optional[bool] = None  # True if super z focus has high accuracy, False otherwise, If None, it means that the super z focus was not used.
```